Added spanish translations [#751]

natastro opened the pull request:
## Changes Need To Be Discussed First

Before opening a pull request, please be sure to announce your changes in [the
developer mailing list](https://www.freelists.org/list/comixed-dev) (and subscribe if you
haven't already). This will help speed up getting your pull request reviewed and merged.

**Will do this today!**

**NOTE:** You can erase any parts of this template that are not applicable to your PR,
including this top section.

Created Spanish translations for all files under 'comixed-web/src/assets/i18n/es'

Commit:
Created Spanish translations

mcpierce commented:
@natastro Please change the commit message to:

```
Added spanish translations [#751]
```

This will link it to the translation ticket for us and meet the commit message convention.

mcpierce commented on comixed-web/src/assets/i18n/es/app.json:
```diff
@@ -1,123 +1,123 @@
+      "import-comics": "Importar cómics",
+      "location-collection": "Ubicaciónes",
+      "publisher-collection": "Editores",
+      "series-collection": "Serie",
```
Should the translation here be "Series" (plural)?

theotocopulitos commented:
Sorry I had to leave the translation here… family health issues keep me tied to real life

Anyways, “Series” is to be translated to “Serie”, not series (that would be plural in Spanish, one “serie”, several “series”)

Cheers

> El 8 jun 2021, a las 15:27, Darryl L. Pierce ***@***.***> escribió:
> 
> ﻿
> @mcpierce requested changes on this pull request.
> 
> In comixed-web/src/assets/i18n/es/app.json:
> 
> > -      "publisher-collection": "Publishers",
> -      "series-collection": "Series",
> -      "story-collection": "Stories",
> -      "team-collection": "Teams",
> -      "unread-comics": "Unread Comics",
> -      "user-preferences": "Preferences",
> -      "web-audit-log": "Web Audit Log"
> +      "all-comics": "Todos los cómics",
> +      "blocked-pages": "Páginas bloqueadas",
> +      "build-details": "Detalles de la compilación",
> +      "character-collection": "Personajes",
> +      "deleted-comics": "Cómics eliminados",
> +      "import-comics": "Importar cómics",
> +      "location-collection": "Ubicaciónes",
> +      "publisher-collection": "Editores",
> +      "series-collection": "Serie",
> Should the translation here be "Series" (plural)?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

mcpierce commented:
> Sorry I had to leave the translation here… family health issues keep me tied to real life Anyways, “Series” is to be translated to “Serie”, not series (that would be plural in Spanish, one “serie”, several “series”) Cheers

This particular entry is the plural since it's the label used for displaying the collection of series in the navigation sidebar (like "Publishers", "Teams", etc.), as opposed to the label used for a single comic's series field.

And your presence is missed, mi amigo. I hope all is well with you and yours.

Commit:
changed serie to series

natastro commented:
Hi @mcpierce and @theotocopulitos ! I changed the title of the PR and changed serie to series. Thank you for your help! Hope all goes well for you soon @theotocopulitos 

Please let me know if you need anything else! ✨

mcpierce commented:
@natastro Thank you for the patch! I'm going to squash it down to a single commit locally and also add your name as a committer! We'll also have some more translation entries coming up for our release at the end of the month and I hope you're available to keep contributing. 😄

mcpierce commented:
@natastro Can you provide a follow up PR for some newer entries that were added for this release?
